docs/static/arcsight-module.asciidoc

```diff
@@ -17,6 +17,16 @@ With a single command, the module taps directly into the ArcSight Smart Connecto
 parses and indexes the security events into Elasticsearch, and installs a suite of Kibana dashboards 
 to get you exploring your data immediately. 
 
+[[arcsight-prereqs]]
+==== Prerequisites
+
+These instructions assume that Logstash, Elasticsearch, and Kibana are already
+installed. The products you need are https://www.elastic.co/downloads[available
+to download] and easy to install. The Elastic Stack 5.6 or later is required for
+this module.   If you are using the Elastic Stack 6.2 and earlier, please see
+the https://www.elastic.co/guide/en/elastic-stack/6.2/index.html[instructions]
```

Comment:
should we have this link directly to the arcsight module docs for 6.2 instead?

https://www.elastic.co/guide/en/logstash/6.2/arcsight-module.html

Reply:
@acchen97 Please clarify.  This topic renders as arcsite-module.html, so I'm not clear on suggested linking.